packages/epo-react-lib/vite.config.ts

```diff
@@ -2,29 +2,96 @@ import { defineConfig } from "vite";
-      name: "EPOLib",
-      formats: ["es", "umd"],
-      fileName: (format) => `epo-lib.${format}.js`,
+      entry: {
+        main: resolve(__dirname, "src/index.ts"),
+        styles: resolve(__dirname, "src/styles"),
+        Accordion: resolve(__dirname, "src/atomic/Accordion/Accordion.tsx"),
+        Button: resolve(__dirname, "src/atomic/Button"),
+        Buttonish: resolve(__dirname, "src/atomic/Buttonish"),
+        CircularLoader: resolve(__dirname, "src/atomic/CircularLoader"),
+        ColorSwatch: resolve(__dirname, "src/atomic/ColorSwatch"),
+        ExpandToggle: resolve(__dirname, "src/atomic/ExpandToggle"),
+        ExternalLink: resolve(__dirname, "src/atomic/ExternalLink"),
+        Figure: resolve(__dirname, "src/atomic/Figure"),
+        Image: resolve(__dirname, "src/atomic/Image"),
+        Link: resolve(__dirname, "src/atomic/Link"),
+        MixedLink: resolve(__dirname, "src/atomic/MixedLink"),
+        ProgressBar: resolve(__dirname, "src/atomic/Progress/Bar"),
+        Marker: resolve(__dirname, "src/atomic/Progress/Marker"),
+        ProgressRadial: resolve(__dirname, "src/atomic/Progress/Radial"),
+        ResponsiveImage: resolve(__dirname, "src/atomic/ResponsiveImage"),
+        Share: resolve(__dirname, "src/atomic/Share"),
+        Toast: resolve(__dirname, "src/atomic/Toast"),
+        Video: resolve(__dirname, "src/atomic/Video"),
+        SimpleTable: resolve(__dirname, "src/content-blocks/SimpleTable"),
+        ComplexTable: resolve(__dirname, "src/content-blocks/ComplexTable"),
+        Error: resolve(__dirname, "src/form/Error"),
+        FormButtons: resolve(__dirname, "src/form/FormButtons"),
+        FormField: resolve(__dirname, "src/form/FormField"),
+        HorizontalSlider: resolve(__dirname, "src/form/HorizontalSlider"),
+        Input: resolve(__dirname, "src/form/Input"),
+        Password: resolve(__dirname, "src/form/Input/patterns/Password"),
+        Select: resolve(__dirname, "src/form/Select"),
+        SelectListbox: resolve(__dirname, "src/form/SelectListbox"),
+        Switch: resolve(__dirname, "src/form/Switch"),
+        BasicModal: resolve(__dirname, "src/layout/BasicModal"),
+        CarouselLayout: resolve(__dirname, "src/layout/Carousel"),
+        Columns: resolve(__dirname, "src/layout/Columns"),
+        Container: resolve(__dirname, "src/layout/Container"),
+        Grid: resolve(__dirname, "src/layout/Grid"),
+        MasonryGrid: resolve(__dirname, "src/layout/MasonryGrid"),
+        SlideoutMenu: resolve(__dirname, "src/layout/SlideoutMenu"),
+        IconComposer: resolve(__dirname, "src/svg/IconComposer"),
+        icons: resolve(__dirname, "src/svg/icons"),
+      },
+      formats: [defaultFormat, "cjs"],
+      fileName: (format, name) =>
+        `${name}.${format === defaultFormat ? "js" : format}`,
     },
     rollupOptions: {
+      plugins: [
+        banner2(({ fileName }) =>
+          fileName.includes("styles") ? `"use client"\n\n` : ""
```

Comment:
sticks `'use client'` on any `styles.ts` file since these will always be client components